build.gradle

```diff
@@ -169,9 +169,9 @@ allprojects {
     test {
         setWorkingDir("$rootDir/")
         useJUnitPlatform()
-        systemProperty 'java.library.path', Paths.get(projectDir.toString(), "iroha_bindings", getCurrentOs()).toString()
+        systemProperty 'java.library.path', Paths.get(projectDir.getParent().toString(), "iroha_bindings", getCurrentOs()).toString()
         println "Runtime Iroha library path:${systemProperties}"
```

Comment:
Can we remove all dependencies related to Ethereum? I don't need them in btc modules.

Reply:
I have moved the root project dependencies further to subprojects (modules)